src/components/generic/movies/movieCard.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import {FC} from 'react'
+import {Icon} from 'react-icons-kit'
+import {star} from 'react-icons-kit/fa/star'
+import {calendar} from 'react-icons-kit/feather'
+import * as Movies from './style'
+
+interface Props {
+  handleOpen(imdb:string):void;
+  poster_med?:string;
```

Comment:
I suggest using camel case for all variable names.

Reply:
Well, since I don't use type generics, This is a response from the API. I can't change it.

Reply:
Gotch, Camel cased them.